src/main/java/com/amazonaws/services/kinesis/scaling/auto/StreamMonitor.java

```diff
@@ -224,15 +224,15 @@ protected ScalingOperationReport processCloudwatchMetrics(
 		// check if we have both get and put votes - if we have both then
 		// implement the decision matrix
 		if (getVote != null && putVote != null) {
-			// if either of the votes are to scale up, then do so. If both are
-			// None,
-			// then do nothing. Otherwise scale down
+			// If either of the votes are to scale up, then do so.
+			// If any vote is DOWN, then scale down.
```

Comment:
@CesarManriqueH The code is designed to scale down only when both getVote **and** putVote are down. However the comment seems to suggest that scaling down happens if either getVote **or** putVote is down. Feel free to clarify if there is a gap in my reading of the statements.
```suggestion
			// If both votes are DOWN, then scale down.
```

Reply:
You're right @sshivananda. Thanks for pointing this out!